doc/source/ray-air/examples/torch_incremental_learning.ipynb

```diff
@@ -512,23 +512,26 @@
    },
    "outputs": [],
    "source": [
-    "from ray.data.preprocessors import BatchMapper\n",
+    "from typing import Dict\n",
+    "import numpy as np\n",
     "\n",
+    "import torch\n",
     "from torchvision import transforms\n",
     "\n",
-    "def preprocess_images(df: pd.DataFrame) -> pd.DataFrame:\n",
-    "    \"\"\"Preprocess images by scaling each channel in the image.\"\"\"\n",
+    "from ray.data.preprocessors import BatchMapper\n",
     "\n",
+    "def preprocess_images(image_batch_dict: Dict[str, np.ndarray]) -> Dict[str, np.ndarray]:\n",
+    "    \"\"\"Preprocess images by scaling each channel in the image.\"\"\"\n",
     "    torchvision_transforms = transforms.Compose(\n",
-    "      [transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))]\n",
+    "      [transforms.Normalize((0.1307,), (0.3081,))]\n",
     "    )\n",
+    "    # Outer dimension is batch size such as (4096, 28, 28)\n",
+    "    image_batch_dict[\"image\"] = torchvision_transforms(\n",
+    "        torch.Tensor(image_batch_dict[\"image\"])\n",
+    "    ).numpy()\n",
```

Comment:
torchvision `ToTensor` also does converts to the range [0,255] and does transposing. do we lose that if we convert to torch tensor manually?

Reply:
what about converting to range [0,255]?